src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -416,6 +412,8 @@ private void purgeUnusedRefDBs()
 	@SuppressWarnings("unchecked")
 	private void createReferences(long personID, Map<String, Map<String, ?>> dbMappings) throws IOException, Exception
 	{
+		String linkCheckReportName = LINK_CHECK_REPORTS_PATH + "/linkCheckSummaryReport" + DateTimeFormatter.ofPattern(DATE_PATTERN_FOR_FILENAMES).format(LocalDateTime.now()) + ".tsv";
+		Files.write(Paths.get(linkCheckReportName), "RefDBName\tNumOK\tNumNotOK\n".getBytes(), StandardOpenOption.CREATE);
```

Comment:
I know it's only two separators, but if more fields were to be added, it might be better and more readable to do:
`String.join("\t", "RefDBName", "NumOK", "NumNotOK")`

Reply:
If it gets longer, I'll probably do that.